app/controllers/doorkeeper/tokens_controller.rb

```diff
@@ -113,19 +113,53 @@ def revoke_token
       # The authorization server responds with HTTP status code 200 if the token
       # has been revoked successfully or if the client submitted an invalid
       # token
-      token.revoke if token&.accessible?
```

Comment:
We have already checked `token.blank?` by the time we call this so the `&.` felt unnecessary, but I can bring it back.